www-servers/lighttpd/lighttpd-1.4.75.ebuild

```diff
@@ -39,7 +39,7 @@ COMMON_DEPEND="
 	ldap? ( >=net-nds/openldap-2.1.26:= )
 	lua? ( ${LUA_DEPS} )
 	maxminddb? ( dev-libs/libmaxminddb )
-	mbedtls? ( net-libs/mbedtls )
+	mbedtls? ( net-libs/mbedtls:0= )
```

Comment:
These changes will need a revbump (`git mv` is fine for this, no keywords change as well).